src/azure-cli/azure/cli/command_modules/storage/_params.py

```diff
@@ -34,6 +34,10 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
     t_queue_service = self.get_sdk('queue#QueueService')
     t_table_service = get_table_data_type(self.cli_ctx, 'table', 'TableService')
 
+    storage_account_type = CLIArgumentType(options_list='--storage-account',
```

Comment:
actually I will prefer keeping using --account-name. I get a typo when I want to use new commands due to convention.

Reply:
we'll use --storage-account and see if the users complain or not to determine if we need also use --account-name.

---
In reply to: [367793774](https://github.com/Azure/azure-cli/pull/11649#discussion_r367793774) [](ancestors = 367793774)